app/components/clock/Clock.jsx

```diff
@@ -33,7 +54,15 @@ class Clock extends React.Component {
     render() {
         return (
             <span className="clock">
-              {this.state.dateNow}
+            <span className="hours">{this.state.hours}:</span>
+            <span className="minutes">{this.state.minutes}:</span>
+            <span className="seconds">{this.state.seconds} </span>
+            {
+                this.props.hourFormat === '12' ? 
```

Comment:
*Ternary* operator is useful when you need to return *one or another* value under different conditions. In your case you need to return value only if your condition is true. This is exact case to use *and* operator (`&&`), like this:

```javascript
this.props.hourFormat === '12' &&
  <span ...
  </span>
```